llvm/utils/TableGen/CodeGenMapTable.cpp

```diff
@@ -491,14 +487,16 @@ void MapTableEmitter::emitTablesWithFunc(raw_ostream &OS) {
 
   // Emit rest of the function body.
   emitMapFuncBody(OS, TableSize);
+
+  OS << "}\n\n";
```

Comment:
moved because opening { for function body is added in this function, so makes sense to add the closing brace here as well.